src/Routes/RosSystemDetail/RosSystemDetail.js

```diff
@@ -36,11 +43,22 @@ class RosSystemDetail extends React.Component {
                 cloud_provider: cloudProvider,
                 instance_type: instanceType,
                 idling_time: idlingTime,
-                rating
+                rating,
+                state: state
```

Comment:
@r14chandra FYI Suggeston: if we directly want to use the object key without changing the name then we can write just `state` here ( just like we did for rating). You can read more about  it here [Rename & Destructure Variables in ES6](https://wesbos.com/destructuring-renaming)